src/app/components/HoldRequest/HoldRequest.jsx

```diff
@@ -86,37 +86,48 @@ class HoldRequest extends React.Component {
       : <p className="loggedInInstruction">Something went wrong retrieving your personal information.</p>;
   }
 
+  renderDeliveryLocation(deliveryLocations = [], callNo) {
+    return deliveryLocations.map((location, i) => (
+      <div key={i} className="group selected">
+        <span className="col location">
+          <a href={`${location.uri}`}>{location['full-name']}</a>
+          <br />{location.address.address1}<br />
```

Comment:
We might need a check here as `location.address.address1` has two layers in the object.